tests/storage/datavolume.go

```diff
@@ -1094,6 +1096,66 @@ var _ = SIGDescribe("DataVolume Integration", func() {
 			Entry("[test_id:5896]with thick provision false, fstrim will make the image smaller", addThickProvisionedFalseAnnotation, true),
 		)
 	})
+
+	Context("VirtualMachinePreference", func() {
+		var vm *v1.VirtualMachine
+		var virtualMachinePreference *instanceType.VirtualMachinePreference
+		var foundSC bool
+		var pvcName string
+
+		BeforeEach(func() {
+			vm, foundSC = tests.NewRandomVMWithDataVolume(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskAlpine), util.NamespaceTestDefault)
+			if !foundSC {
+				Skip("Skip test when Filesystem storage is not present")
+			}
```

Comment:
Are you using this because we know it's using a storageclass? I wonder if we just need to clone an existing storage class and rename it here for our tests given we only want to assert the name that ends up in the `VirtualMachine`.

Reply:
I think it should be enough just to check the name of storage class without creating it.